### ISSUE TITLE
More test results

### ISSUE THREAD

**Issue by LZL0** [JohnSully/KeyDB#44]
Ubuntu 18.04 LTS - Kernel v5.1.9

Providing only the relevant part:
```
[34/52 done]: unit/type/list-3 (48 seconds)
Testing unit/wait
[err]: FLUSHDB ASYNC can reclaim memory in background in tests/unit/lazyfree.tcl
Expected condition '$peak_mem > $orig_mem+1000000' to be true (13838096 > 13734512+1000000)
```

**Comment by LZL0** [comment]
Should I be concerned in production about this warning?

**Comment by JohnSully** [comment]
Tl;dr no need to worry, the explanation is below:

The memory tests can be unreliable because prior tests run on the same instance can influence memory consumption. The test suite does not spawn a new instance for every test because it would dramatically increase the running time. 

In this case your failure is saying that the test consumed too little memory and therefore it’s results are inconclusive. 

I get this failure intermittently from time to time and have manually verified background free is working as expected. So no need to worry. 

Improving memory test reliability is a work item on my backlog.

**Comment by LZL0** [comment]
Thank you for your detailed answer! :)
